.github/workflows/release-build.yml

```diff
@@ -34,4 +34,4 @@ jobs:
         run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/v}
 
       - name: Publish
-        run: make IMAGE_TAG=$VERSION publish
\ No newline at end of file
+        run: make VERSION=$VERSION publish
```

Comment:
I wonder if this is needed? :)

Reply:
Hm, I assume you would only ever trigger this flow **after** running `make release`? If so, I can remove the tag from this command.

Reply:
Actually, I think this will still be needed as `make release` won't replace the `VERSION` inside Makefile. So you'll still need to specify the version, otherwise it will publish the default of `master`.